manager/controlapi/ca_rotation_test.go

```diff
@@ -290,7 +290,7 @@ func TestValidateCAConfigInvalidValues(t *testing.T) {
 			expectErrorString: "certificate key mismatch",
 		},
 		{
-			// this is just one class of failures caught by NewRootCA, not going to bother testing others, since they are
+			// this is just one class of failures caught by NewRootCA, not going to bother to test others, since they are
```

Comment:
I think both ways is actually correct (and "don't bother doing something" may actually be more common)